references/V1_OutboundCallback.md

```diff
@@ -265,6 +267,24 @@ Send back the verification challenge to show that the webhook supports this call
 |**response**  <br>*required*|The challenge that was provided in the request body of the verification request.|string|
 
 
+<a name="addreactionbody"></a>
+### AddReactionBody
```

Comment:
Can these be formatted in the same way (using a table) as the rest of the other data structures for events?
Seems that the actual elements of the structure are missing.  @nivekselig I believe you will be replacing this file based on output of build which is based on the .yml counterpart.  IOW, this .md file is build output and should not be part of this PR.

Reply:
Left this since it will get regenerated when build is done, anyway.